### PR TITLE
fix automation to work with new kata plugin image

### PR DESCRIPTION
**Pull request by pruan-rht**
@liangxia @cpmeadors @valtair1891 PTAL

### CODE REVIEW

File: testdata/kata/release-4.8/deployment.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 ---
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   name: kata-operator
-  namespace: sandboxed-containers-operator-system
+  namespace: openshift-sandboxed-containers-operator
 spec:
-  channel: "4.8"
+  channel: "1.0"
```
Comment (liangxia) [inline testdata/kata/release-4.8/deployment.yaml:21]: In above code, `  cb.channel = cb.master_version`, then
```
    step %Q/I perform the :set_custom_channel_and_subscribe web action with:/, table(%{
      | update_channel    | #{cb.channel} |
      | install_mode      | OwnNamespace  |
      | approval_strategy | Automatic     |
```
Which one are we expecting ?

Comment (pruan-rht) [reply testdata/kata/release-4.8/deployment.yaml:21]: in the old way it was following OCP version.  I'm getting it from the subscription now.  I'll take that old line out.

Comment (pruan-rht) [reply testdata/kata/release-4.8/deployment.yaml:21]: fixed

**Pull request merged**